docs/run_test/platform.md

```diff
@@ -0,0 +1,163 @@
+    subscription to authenticate your current session. LISA also supports other
+    Azure authentications, refer to [runbook reference](runbook.md).
+
+    Here, let's choose `Azure CLI` for the setup. You should see a page pop up
+    and all your Azure subscriptions shown in console.
+
+    ```bash
+    az login
+    ```
+
+2. Get the subscription id
+
+    LISA needs to know the Azure subscription ID for your testing. Run below
+    command to retrieve subscription information.
+
+    ```bash
+    az account show --subscription "<your subscription Name>"
+    ```
+
+    For now you only need the `<subscription id>` for future use.
+
+    ```json
+    {
+        "environmentName": "AzureCloud",
+        "homeTenantId": "<tenant id>",
+        "id": "<subscription id>",
+        "isDefault": true,
+        "managedByTenants": [],
+        "name": "<subscription name>",
+        "state": "Enabled",
+        "tenantId": "<tenant id>",
+        "user": {
+            "name": "<user account>",
+            "type": "user"
+        }
+    }
+    ```
+
+    AzureCloud in this example is set to default. You do not need to do that as
+    long as you provide the correct Azure subscription ID.
+
+3. Prepare SSH key pair
+
+    LISA connects to the Azure test VM by SSH with key authentication; please
+    have your key pair (public key and private key) ready before running the
+    test. If you don't have a key pair, run below command to create a new one.
+
+    ```bash
+    ssh-keygen
+    ```
+
+    :warning:   Don't use passphrase to protect your key. LISA doesn't support
+    that.
+
+4. Run LISA
+
+    Use above `<subscription id>` and `<private key file>` to run LISA. It might
+    take several minutes to complete.
+
+    ```bash
+    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>"
+    ```
+
+5. Verify test result
+
+    After test completed, you can check the LISA console log, or the html report
+    file for the test results. See an example html report as below:
+
+    ![image](img/smoke_test_result.png)
+
+    If you want to customize your test using more specified parameters, please
+    refer to [runbook reference](runbook.md) for more information.
+
+### Run in Ready computers
+
+If you have prepared a Linux computer for testing, please run LISA with `ready`
+runbook:
+
+1. Get the IP address of your computer for testing.
+
+2. Get the SSH public/private key pair which can access this computer.
+
+3. Run LISA with parameters below:
+
+    ```bash
+    lisa -r ./microsoft/runbook/ready.yml -v public_address:<public address> -v "user_name:<user name>" -v "admin_private_key_file:<private key file>"
+    ```
+
+`ready` runbook also supports tests which require multiple computers (for
+example, networking testing); and, it supports password authentication too.
+Learn more from [runbook reference](runbook.md).
+
+## Run Microsoft tests
```

Comment:
Because this file focuses on platform, we don't need to mention about tiers. But we may need a place to introduce the variables can be used together.